src/RecoveryServices/RecoveryServices.SiteRecovery.Autorest/custom/Cmdlets/Fabric/Get-AzRecoveryServicesReplicationFabric_GetByName.ps1

```diff
@@ -0,0 +1,132 @@
+        # The name of the resource group where the recovery services vault is present.
+        ${ResourceGroupName},
+
+        [Parameter(Mandatory)]
+        [Microsoft.Azure.PowerShell.Cmdlets.RecoveryServices.Category('Path')]
+        [System.String]
+        # The name of the recovery services vault.
+        ${ResourceName},
+
+        [Parameter()]
+        [Microsoft.Azure.PowerShell.Cmdlets.RecoveryServices.Category('Path')]
+        [Microsoft.Azure.PowerShell.Cmdlets.RecoveryServices.Runtime.DefaultInfo(Script='(Get-AzContext).Subscription.Id')]
+        [System.String[]]
+        # The subscription Id.
+        ${SubscriptionId},
+
+        [Parameter()]
+        [Microsoft.Azure.PowerShell.Cmdlets.RecoveryServices.Category('Query')]
+        [System.String]
+        # OData filter options.
+        ${Filter},
+
+        [Parameter()]
+        [Alias('AzureRMContext', 'AzureCredential')]
+        [ValidateNotNull()]
+        [Microsoft.Azure.PowerShell.Cmdlets.RecoveryServices.Category('Azure')]
+        [System.Management.Automation.PSObject]
+        # The credentials, account, tenant, and subscription used for communication with Azure.
+        ${DefaultProfile},
+
+        [Parameter(DontShow)]
+        [Microsoft.Azure.PowerShell.Cmdlets.RecoveryServices.Category('Runtime')]
+        [System.Management.Automation.SwitchParameter]
+        # Wait for .NET debugger to attach
+        ${Break},
+
+        [Parameter(DontShow)]
+        [ValidateNotNull()]
+        [Microsoft.Azure.PowerShell.Cmdlets.RecoveryServices.Category('Runtime')]
+        [Microsoft.Azure.PowerShell.Cmdlets.RecoveryServices.Runtime.SendAsyncStep[]]
+        # SendAsync Pipeline Steps to be appended to the front of the pipeline
+        ${HttpPipelineAppend},
+
+        [Parameter(DontShow)]
+        [ValidateNotNull()]
+        [Microsoft.Azure.PowerShell.Cmdlets.RecoveryServices.Category('Runtime')]
+        [Microsoft.Azure.PowerShell.Cmdlets.RecoveryServices.Runtime.SendAsyncStep[]]
+        # SendAsync Pipeline Steps to be prepended to the front of the pipeline
+        ${HttpPipelinePrepend},
+
+        [Parameter(DontShow)]
+        [Microsoft.Azure.PowerShell.Cmdlets.RecoveryServices.Category('Runtime')]
+        [System.Uri]
+        # The URI for the proxy server to use
+        ${Proxy},
+
+        [Parameter(DontShow)]
+        [ValidateNotNull()]
+        [Microsoft.Azure.PowerShell.Cmdlets.RecoveryServices.Category('Runtime')]
+        [System.Management.Automation.PSCredential]
+        # Credentials for a proxy server to use for the remote call
+        ${ProxyCredential},
+
+        [Parameter(DontShow)]
+        [Microsoft.Azure.PowerShell.Cmdlets.RecoveryServices.Category('Runtime')]
+        [System.Management.Automation.SwitchParameter]
+        # Use the default credentials for the proxy
+        ${ProxyUseDefaultCredentials}
+    )
+
+    process {
+        try {
+            $null = $PSBoundParameters.Remove("FriendlyName")
```

Comment:
Are we just doing it by friendly name and not fabric name  ?

Reply:
We have 2 commands one doing with only fabric name and another doing only with friendly name.

Reply:
No, let's not have two commands, we should have one single command with different parametersets like we have in existing module.